google/cloud/bigquery/table.py

```diff
@@ -1897,11 +1897,6 @@ def total_bytes_processed(self) -> Optional[int]:
         """total bytes processed from job statistics, if present."""
         return self._total_bytes_processed
 
-    @property
-    def page_size(self) -> Optional[int]:
-        """The maximum number of rows in each page of results from this request, if present."""
-        return self._page_size
-
```

Comment:
This would technically be a breaking change, but maybe it's OK since this property was only present in one version?